src/screens/TopicPageScreen/index.js

```diff
@@ -398,10 +403,17 @@ const TopicPageScreen = (props) => {
     />
   );
 
-  if (isInitialLoading) return null;
   return (
-    <SafeAreaProvider forceInset={{top: 'always'}} style={styles.parentContainer}>
+    <>
       <NavHeader
+        initialData={{
+          channelPicutre: params.channelPicture,
+          coverImage: params.coverImage,
+          isFollowing: params.isFollowing,
+          memberCount: params.memberCount
+        }}
+        isLoading={isInitialLoading}
+        opacityImage={opacityImage}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The changes here seem to be related to the loading state of the `NavHeader` component. The new code now passes an `isLoading` prop and an `initialData` prop to `NavHeader`, which can be useful for displaying a loading state or initial data. However, ensure that the `NavHeader` component properly handles these new props.
```javascript
<NavHeader
  initialData={{
    channelPicutre: params.channelPicture,
    coverImage: params.coverImage,
    isFollowing: params.isFollowing,
    memberCount: params.memberCount
  }}
  isLoading={isInitialLoading}
  ...
/>
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->